crypto/evp/evp_lib.c

```diff
@@ -333,6 +333,16 @@ int EVP_CIPHER_CTX_iv_length(const EVP_CIPHER_CTX *ctx)
     return v;
 }
 
+int EVP_CIPHER_CTX_tag_length(const EVP_CIPHER_CTX *ctx)
+{
+    int ret, v;
+    OSSL_PARAM params[2] = { OSSL_PARAM_END, OSSL_PARAM_END };
+
+    params[0] = OSSL_PARAM_construct_int(OSSL_CIPHER_PARAM_AEAD_TAGLEN, &v);
+    ret = evp_do_ciph_ctx_getparams(ctx->cipher, ctx->provctx, params);
```

Comment:
Why not just call `EVP_CIPHER_CTX_get_params()` here?
Perhaps also have a fallback to `EVP_CIPHER_get_params()` if the context one isn't defined.

Reply:
It does the same as all the other calls...